adal/build.gradle

```diff
@@ -78,6 +79,40 @@ dependencies {
     testCompile 'junit:junit:4.12'
 }
 
+
+task createPom  {
+    pom {
+        project {
+            groupId 'com.microsoft.aad'
+            artifactId 'adal-aggregator'
```

Comment:
>artifactId 'adal-aggregator' [](start = 12, length = 28)

check what do we have today please. [http://search.maven.org/#search%7Cga%7C1%7Caad](http://search.maven.org/#search%7Cga%7C1%7Caad)
this will make adal released to a completely different artifact.